Decisions/knowledge-management/information-classification.md

```diff
@@ -0,0 +1,54 @@
+# Information Classification Levels
+
+| Status      | draft                                            |
+| :---------- | :-------------------------------------------------- |
+| **PR #**    | [16](https://github.com/blindnet-io/openness/pull/16) |
+| **Sponsor** | Filip Radulovic (filip@blindnet.io)                        |
+
+## (-2) Public
+
+Promoted information
+
+Examples:
+
+- blindnet blog post
+- any information on blindnet.io
+- any content referenced by a link in blindnet.io
+- any content shared on social media
+
+## (-1) Open
+
+Information anyone could access online via an URL.
+
+Example:
+
+- unpromoted Github repository
+- unreferenced video on Youtube
+- Google document shared with "anyone with the link"
+
+## (0) _Unclassified_
+
+> Default level
+
+Information accessible without extensive efforts.
```

Comment:
Yeah, my idea here is to say: you may be thinking you have good enough control over this data, but you're certainly wrong because you didn't take actual measures to protect it + we don't track these data, so please be careful to not put sensitive information about blindnet there. "Without extensive efforts" means protections can be "cracked" easily.